### ISSUE TITLE
AttributeError: 'Quantity' object has no 'rms_weighted' member

### ISSUE THREAD

**Issue by matteobachetti** [nanograv/PINT#887]
I'm having this problem with pintk, that might or might not be associated with some library version
I tried Astropy 4.2 and 4.0.

To reproduce:
```
$ pintk testtimes.par testtimes.tim

INFO: IERS B Table appears to be old. Attempting to re-download. [pint.erfautils]
Downloading http://hpiers.obspm.fr/iers/eop/eopc04/eopc04_IAU2000.62-now
|===================================================| 3.3M/3.3M (100.00%)         5s

(...)

arecibo TOAs (4):
  Min freq:      1500.000 MHz 
  Max freq:      1500.000 MHz 
  Min error:     2.94 us
  Max error:     2.94 us
  Median error:  2.94 us
gbt TOAs (4):
  Min freq:      1500.000 MHz 
  Max freq:      1500.000 MHz 
  Min error:     2.94 us
  Max error:     2.94 us
  Median error:  2.94 us

Traceback (most recent call last):
  File "/Users/meo/anaconda3/envs/py3/bin/pintk", line 33, in <module>
    sys.exit(load_entry_point('pint-pulsar', 'console_scripts', 'pintk')())
  File "/Users/meo/devel/spyder_projects/PINT/src/pint/scripts/pintk.py", line 172, in main
    app = PINTk(root, parfile=args.parfile, timfile=args.timfile, ephem=args.ephem)
  File "/Users/meo/devel/spyder_projects/PINT/src/pint/scripts/pintk.py", line 49, in __init__
    self.openPulsar(parfile=parfile, timfile=timfile, ephem=ephem)
  File "/Users/meo/devel/spyder_projects/PINT/src/pint/scripts/pintk.py", line 118, in openPulsar
    self.psr = Pulsar(parfile, timfile, ephem)
  File "/Users/meo/devel/spyder_projects/PINT/src/pint/pintk/pulsar.py", line 99, in __init__
    % self.prefit_resids.time_resids.rms_weighted().to(u.us).value
  File "/Users/meo/anaconda3/envs/py3/lib/python3.7/site-packages/astropy/units/quantity.py", line 835, in __getattr__
    attr))
AttributeError: 'Quantity' object has no 'rms_weighted' member

```

**Comment by matteobachetti** [comment]
@paulray @aarchiba : do you have this issue? What Astropy version(s) are you using?

**Comment by matteobachetti** [comment]
Resolved in #884

**Comment by paulray** [comment]
Sorry about that! pintk is not tested as part of the CI testing so a recent change broke it.  Thanks for the fix!
